pkg/cluster/deploybaseresources.go

```diff
@@ -19,10 +19,10 @@ import (
 	utilrand "k8s.io/apimachinery/pkg/util/rand"
 
 	"github.com/Azure/ARO-RP/pkg/api"
+	"github.com/Azure/ARO-RP/pkg/api/util/subnet"
```

Comment:
I noticed that in all of the other places that refer to the new `pkg/api/util/subnet`, you renamed the import to `api_subnet`, but not here in `pkg/cluster`. Why is that?

Reply:
I think my IDE put that in :D